bridge/hmd/openvr.py

```diff
@@ -0,0 +1,136 @@
+"""
+OpenVR
+======
+
+OpenVR Compatible head mounted display
+It uses a python wrapper to connect with the SDK
+
+"""
+
+from . import HMD as baseHMD
+
+import glew32 as glew
+import openvr_api as openvr
+import bridge_wrapper as bridge
+
+import ctypes
+
+from ctypes import (
+        c_float,
+        )
+
+class HMD(baseHMD):
+
+    def init_ctypes(self):
+        """  I spent a day getting access violations.. I don't know how dalai's just seems to work but this one doesn't work without explicity setting the return types... baffled.
```

Comment:
Great :).  I was scratching my head on that one and thought there was something I was missing because the python side was treating memory addresses as integers..  Didn't realise it was a 64/32 bit difference and that it would fix your 64 bit issue so I'm glad to have stumbled on that fix :)